TestAssets/TestProjects/PerformanceTestProjects/TwoTargetGraphLarge/TwoTargetLargeP4/project.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
For the multi-project apps, is it intentional not to add actual code dependencies?  I would recommend actually calling code in the dependencies so that you get representative build time.